five-minute-serverless-backend/README.md

```diff
@@ -1,5 +1,5 @@
-# Five Minute Serverless Backend
-*How to build a **serverless backend** for your web applciation and have it up in running on the cloud in minutes.*
+# Five Minutes Serverless Backend
```

Comment:
It's actually correct as is. Using the singular is valid when the thing being described is singular "Serverless Backend". Here is a coincidentally perfect reference https://ell.stackexchange.com/questions/14290/shouldnt-five-minute-walk-be-five-minutes-walk-in-this-sentence